enhancements/update/change-management-and-maintenance-schedules.md

```diff
@@ -0,0 +1,1158 @@
+1. Curious, the service consumer checks the projects ClusterVersion within the HostedCluster and reviews its `status` stanza. It shows that changes are pending and the time of the next window in which changes can be initiated.  
+1. Separate metrics specific to change management indicate that changes are pending for both resources.
+1. The non-Red Hat operations team has alerts setup to fire when changes are pending and the number of 
+    seconds before the next permitted window is less than 2 days away.
+1. These alerts fire after Thursday UTC 00:00 to inform the operations team that changes are about to be applied to the control-plane.
+1. It is not the first week of the month, so there is no alert fired for the NodePool pending changes.
+1. The operations team is comfortable with the changes being rolled out on the control-plane. 
+1. On Saturday 00:00 UTC, the HyperShift operator initiates changes the control-plane update.
+1. The update completes without issue.
+1. Changes remain pending for the NodePool resource.
+1. As the first Saturday of the month approaches, the operations alerts fire to inform the team of forthcoming changes.
+1. The operations team realizes that a corporate team needs to use the cluster heavily during the weekend for a business critical deliverable.
+1. The service consumer logs into OCM and adds an exclusion for the upcoming Saturday.
+1. Interpreting the new exclusion, the metric for time remaining until a permitted window increases to count down to the following month's first Saturday.
+1. A month passes and the pending cause the configured alerts to fire again.
+1. The operations team is comfortable with the forthcoming changes.
+1. The first Saturday of the month 00:00 UTC arrives. The HyperShift operator initiates the worker-node updates based on the pending changes in the cluster NodePool.
+1. The HCP cluster has a large number of worker nodes and draining and rebooting them is time-consuming. 
+1. At 23:59 UTC Saturday night, 80% of worker-nodes have been updated. Since the maintenance schedule still permits the initiation of material changes, another worker-node begins to be updated.
+1. The update of this worker-node continues, but at 00:00 UTC Sunday, no further material changes are permitted by the change management policy and the worker-node update process is effectively paused.
+1. Because not all worker-nodes have been updated, changes are still reported as pending via metrics for NodePool.  **TODO: Review with HyperShift. Pausing progress should be possible, but a metric indicating changes still pending may not since they interact only through CAPI.**
+1. The HCP cluster runs with worker-nodes at mixed versions throughout the month. The N-1 skew between the old kubelet versions and control-plane is supported.
+1. **TODO: Review with Service Delivery. If the user requested another minor bump to their control-plane, how does OCM prevent unsupported version skew today?**
```

Comment:
We should have skew protection (whatever it's called) disabled because we'll force a minor version upgrade of control planes in ROSA HCP when the control plane version is EOL.  Validation of allowed skew is possible at other layers like OCM/CS, not sure what we permit at this time.